package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "selenium-standalone",
-  "version": "8.0.10",
+  "version": "8.0.11",
```

We have an automated way to update the version
```suggestion
  "version": "8.0.10",
```